monitoring/ceph-mixin/alerts.jsonnet

```
@@ -0,0 +1 @@
+std.manifestYamlDoc((import 'alerts.libsonnet').prometheusAlerts)
```

To generate a human-friendly YAML (no clue why the jsonnet defaults are the other way around):
```suggestion
std.manifestYamlDoc((import 'alerts.libsonnet').prometheusAlerts, indent_array_in_object=true,  quote_keys=false)
```
I also hope this will generate an easier to read diff too.